docs/rfcs/1003-mandatory-human-handle.md

```diff
@@ -0,0 +1,164 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# Mandatory HumanHandle & DeviceLabel fields
+
+## Basic considerations
+
+Human handle field in user certificate is optional.
+Device label in device certificate is also optional.
+
+This is due to multiple reasons:
+
+### 1 - Legacy reason
+
+Those fields have been introduced in Parsec v1.13.
+Hence all certificates created with an older version lack those fields.
+
+### 2 - Support non-human user reason
+
+The need for human handle appeared from the fact a human wants to be able to
+revoke his current account (i.e. his user in Parsec jargon) and re-create one
+(typically in case the account is compromised) while retaining his identity
+(e.g. user id "Gorge" is already taken by a now revoked user, hence you'll be
+known as "Jorge" from now on...).
+
+On the other hand a non-human account are fine with changing there id (typically
+revoking `sgx-anti-virus-20230901` and replacing it by `sgx-anti-virus-20230902`).
+
+However in practice we never used non-human accounts.
+On top of that the human handle field doesn't prevent from creating non-human account:
+the email field can be set to a dummy value (or event to a valid address to contact
+the team responsible for this).
+
+### 3 - Hide identity from outsiders
+
+The Outsider profile is to be use for user that should be allowed to work in
+an organization while not being trusted enough to know the full identity of
+who is part of said organization.
```

Comment:
```suggestion
who is a part of said organization.
```

Reply:
Event better <https://www.linguee.fr/francais-anglais/search?source=auto&query=who+is+part+of>